src/core/i18n/en/translation.en.json

```diff
@@ -1974,6 +1978,7 @@
         }
       },
       "innovation-packs": {
+        "create": "Create InnovationPack",
```

Comment:
A space between "Innovation"  and "Pack", I guess.

Reply:
It's because Rene's message :)